.github/workflows/imaginary.yml

```diff
@@ -11,27 +11,63 @@ jobs:
     name: Create Imaginary Container
 
     steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+        with:
+          repository: h2non/imaginary
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+        with:
+          platforms: all
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          driver-opts: env.BUILDKIT_STEP_LOG_MAX_SIZE=-1

```

Comment:
is this really needed?

Reply:
Do you mean the `driver-opts: env.BUILDKIT_STEP_LOG_MAX_SIZE=-1` part? If yes, yes because if the build logs are getting too big, they will be hidden and this disables it.

Reply:
Okay